webapp/app/config.py

```diff
@@ -57,6 +58,7 @@ class ProductionConfig(BaseConfig):
 
 
 class StagingConfig(BaseConfig):
+    ALLOW_TESTING_ROUTES = True
```

Comment:
Ok, last annoying question: should we maybe disable this in Staging, too? That's our "production-like-but-not-production" environment and we should probably keep it as close as possible to production as we can. What's more: there should never be a need to have this enabled in Staging as - almost by definition - any acceptance tests should drive the app just like user would, filling in form fields and submitting them. WDYT?

Reply:
Yes, that makes sense